features/step_definitions/purap/requisition.rb

```diff
@@ -1,6 +1,23 @@
-And /^I create a Requisition with required Delivery and Additional Institutional information populated$/ do
+And /^I create a Requisition with required Chart-Organization, Delivery and Additional Institutional information populated$/ do
   visit(MainPage).requisition
   on RequisitionPage do |req_page|
+    # Check Chart/Org associated with the user; if either one does not exist, use corresponding parameter default value and perform lookup
+    chart,org = req_page.chart_org_readonly.split('/')
+    (chart.nil? || chart.empty?) ? (nil) : (chart = chart.strip())
+    (org.nil? || org.empty?) ? (nil) : (org = org.strip())
+    if ((org.nil? || org.empty?) || (chart.nil? || chart.empty?))
```

Comment:
You cannot just check for nil at this point.  Lines 6 and 7 are present to deal with leading-trailing white space as well as either data value initially being nil when it is pulled in from the page. The call to method strip on lines 6 and 7 will create a zero length (empty) string if either values is all white space not a nil.  So, if you only check for nil and this point, the zero length string created by the call to strip will be missed and the data will not be fixed as it needs to be.